### DISCUSSION TITLE
Copilot

### DISCUSSION THREAD

**Discussion by gavinowens** [community/community#19057] [GitHub Education] [answered]
I read that students get access to copilot? How do I access that? Currently it just brings me to the purchase screen when I click on Copilot. It looks really cool.

**Comment by patricklarose** [comment]
You have to apply for a student developer pack. Once you are approved you will be able to use Copilot.

Here's a guide on Github Docs on how to apply:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack

**Reply by gavinowens** [reply]
I have a Student Developer Pack. I got one while I was doing virtual school last year. I still show as having Pro on my dashboard.

**Reply by nasso** [reply]
I have a student developer pack too but I am brought to the purchase screen as well.

I couldn't find Copilot in the list of offers in the Student Developer Pack: https://education.github.com/pack/offers

**Reply by cheesycod** [reply]
Me neither, I thought it was free as well

**Reply by davidleejy** [reply]
Same here. Got access to the student developer pack but copilot still wants to charge. Subscription page does not recognize student status.

![Screenshot 2022-06-22 at 1 35 17 AM](https://user-images.githubusercontent.com/1690072/174866884-cf87ac11-6528-4e00-95e6-162768f3047d.png)


![Screenshot 2022-06-22 at 1 54 55 AM](https://user-images.githubusercontent.com/1690072/174867080-727238d5-c8ba-4413-a3fb-be1d552235a3.png)

**Reply by ldeluigi** [reply]
same

**Reply by Ethan707** [reply]
Same issue

**Reply by nasso** [reply]
Please do not reply just to say "same here", an upvote can convey the exact same information without sending a push notification to those who subscribed to this discussion for actual updates.

**Reply by Avenred** [reply]
What worked for me was going to the ["Get your GitHub student benefits" page](https://education.github.com/discount_requests/student_application), and re-verifying that I was a student. If you see a yellow banner that says, "It is not necessary for you to re verify at this time." then I don't think this method will work.
Once you've re-verified that you're a student, visiting the copilot website and clicking the free trial button should give you copilot for free.
I might have gotten lucky here, since I haven't verified that I was a student since 2020, but I still had pro status on my account and whatnot when I went to re-verify today.

**Comment by ldeluigi** [comment, marked as answer]
Duplicate of #19050?

**Reply by Codooze** [reply]
and there is the solution https://github.com/github-copilot/free_signup I just visited this link and now I have copilot active

**Reply by gavinowens** [reply]
> and there is the solution https://github.com/github-copilot/free_signup I just visited this link and now I have copilot active

Thank you! I can't wait to try Copilot!

**Reply by vixniv** [reply]
> and there is the solution https://github.com/github-copilot/free_signup I just visited this link and now I have copilot active

It works, thank you so much

**Reply by iamgoodbytes** [reply]
I'm a github campus advisor since a couple of years and teach github to my students in college. I've used co-pilot and have taught my students how to take advantage of it as well, but I can't figure out a way to continue doing so. Is there anything I can do to activate co-pilot as a teacher/campus advisor on my account?

**Reply by Anthony9811** [reply]
Not working for me sadly, it still asks me to pay

**Reply by amritpal7** [reply]
The link is redirecting to the payment page...

**Reply by amritpal7** [reply]
Do we need to enter the payment details to have the student access ?

**Reply by gavinowens** [reply]
> Do we need to enter the payment details to have the student access ?

I didn't have to.

**Reply by nazmus-ashrafi** [reply]
Thanks, that worked!

**Reply by kevinfinalboss** [reply]
thanks, it solved my problem

**Comment by imhalid** [comment]
You must add an e-mail address ending in .edu to your account. You must then submit a document proving that you are a student.

**Reply by trodrigueza** [reply]
I think most of us wondering about this issue already have the student github pro plan.

**Reply by imhalid** [reply]
> Since I did not encounter this problem, I do not know about the solution, unfortunately.

**Reply by gavinowens** [reply]
I have the student plan already. The docs say that Copilot is free for students yet it brings me to the purchase page.

**Reply by danielolaviobr** [reply]
> I have the student plan already. The docs say that Copilot is free for students yet it brings me to the purchase page.

Same here, I was in the beta, not sure if this affects it.

**Comment by colossalchicken** [comment]
Try settings > Code, planning, and automation > copilot

**Reply by gavinowens** [reply]
Thanks, but @jeison-AK figured it out.

**Comment by ghost** [comment]
Hh
